### ISSUE TITLE
Append file is not append

### ISSUE THREAD

**Issue by kevinjmh** [Alluxio/alluxio#8212]
The append function required the file does not exists, and it returns FSDataOutputStream from a new file. This is a New File action instead of append action.

https://github.com/Alluxio/alluxio/blob/21a3a87747010f087d1937f48bba3caa883885f8/core/client/hdfs/src/main/java/alluxio/hadoop/AbstractFileSystem.java#L127-L138

**Comment by apc999** [comment]
@kevinjmh  correct. File Append is not currently supported in Alluxio. Do you have a good use case? We didn't really prioritize this feature because we haven't seen enough use cases for append operations. Sharing your target cases will help us re-prioritize work. Thanks!
